### ISSUE TITLE
KP.3.1.1 + Orf1b:S1658Y with Orf9b:P39L acquired convergently S:W64C  (17 , 5 Places) one recent from Brazil with S:N481R

### ISSUE THREAD

**Issue by FedeGueli** [sars-cov-2-variants/lineage-proposals#2097]
Another S:W64C lineage on Kp.3.1.1 backbone after #2028

KP.3.1.1 >> Orf1b:S1658Y (C18440A) > ORF9b:P39L (C28399T) > Orf1a:N4315T (A13209C), C19572T, G21754T
New query: A13209C, G21754T,C19572T 
Samples: 17
First collected on 2024-06-02 in Spain
Last collected on 2024-09-05 in Brazil  
Places: 5 US, Spain, France (GBW), Ireland, Brazil
Tree:
<img width="1170" alt="Screenshot 2024-09-27 alle 10 52 12" src="https://github.com/user-attachments/assets/53de6817-497d-4a67-a7d5-013cd7b381cf">
https://nextstrain.org/fetch/genome-test.gi.ucsc.edu/trash/ct/subtreeAuspice39_genome_test_5710c_65ccf0.json?c=userOrOld&label=id:node_7193199

Interestingly the Brazilian samples carries S:N481R according to Nextclade and Gisaid (USher doesnt recognize it)

**Comment by aviczhl2** [comment]
Duplicate of #1892

**Comment by FedeGueli** [comment]
Thank you.
